### DISCUSSION TITLE
Help I'm new my station runs on VLC but not on winamp

### DISCUSSION THREAD

**Discussion by cybercuate** [AzuraCast/AzuraCast#5378] [Q&A]
I installed shoutcast 2.6.1 (because everything is free again) and the proxy works ok but take a look at:
https://server.stereoanime.net/radio/8000/radio
Q1 Why VLC or web browser can play it but winamp can't? and VLC doesn't show any metadata.
Also, can I make the URL less redundant (radio/8000/radio)? 
Q2
https://server.stereoanime.net:8000/radio doesn't work but
https://123.45.67:8000/radio works (even winamp can play it and shows metadata), why?

My DNS is managed by cloudflare
winamp5.8

**Comment by BusterNeece** [comment]
> I installed shoutcast 2.6.1 (because everything is free again)

Wait what?

**Reply by cybercuate** [reply]
> > I installed shoutcast 2.6.1 (because everything is free again)
> 
> Wait what?

http://forums.winamp.com/showthread.php?t=453775&page=2
http://forums.winamp.com/showthread.php?t=453775

**Comment by cybercuate** [comment]
> I installed shoutcast 2.6.1 (because everything is free again) and the proxy works ok but take a look at: https://server.stereoanime.net/radio/8000/radio Q1 Why VLC or web browser can play it but winamp can't? and VLC doesn't show any metadata. Also, can I make the URL less redundant (radio/8000/radio)? Q2 https://server.stereoanime.net:8000/radio doesn't work but https://123.45.67:8000/radio works (even winamp can play it and shows metadata), why?
> 
> My DNS is managed by cloudflare winamp5.8

nevermind I did it. It was cloudflare, I just had to change the "proxy(orange cloud)" to "DNS only(gray cloud)" and that was it. Now I can go to https://server.stereoanime.net:8000 even on chrome based browsers which are known to block anything that is not https.
